lib/js/stanford_publication.behavior.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Behavior Example that works with Webpack.
+ *
+ * @see: https://www.npmjs.com/package/drupal-behaviors-loader
+ *
+ * Webpack wraps everything in enclosures and hides the global variables from
+ * scripts so special handling is needed.
+ */
+
+export default {
+
+  // Attach Drupal Behavior.
+  attach: function (context, settings) {
+    (function ($) {
+
+      // We want to only open and close filter menu when on responsive sizes.
+      const mediaQuery = window.matchMedia('(max-width: 991px)');
+
+      function handleTabletChange(e) {
+        // Check if the media query is true
+        if (e.matches) {
+
+          // Replace the h2 with button for responsive views.
+          const buttonLabel = $("#system-menu-blockstanford-publication-topics-menu").text();
+
+          $("#system-menu-blockstanford-publication-topics-menu")
+            .replaceWith('<button class="publication-topics__collapsable-menu">' + buttonLabel + '</button>');
```

Comment:
I don't really like the replace with. the wrap was a better approach. is there a reason you used this method?

Reply:
I did it because I couldn't get the unwrap to work properly and because I was essentially replacing the `H2`.

Reply:
I'm not really sure why you're replacing the `h2` though. wrapping an `h2` with a `button` shouldn't really cause any issue.